CONTRIBUTING.md

```diff
@@ -211,6 +229,50 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 - 修复文档页面的渲染或显示错误。
 - 修复文档内的死链。
 
+</details>
+
+<details>
+<summary>Markdown 规范</summary>
+
+TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量和格式规范，你修改的文档需要遵循一定的 Markdown 规则。我们为 docs-cn 仓库设置了检测 markdown 文件规范的 CI check，即 [markdownlint check](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md)。如果你提交的 PR 不符合规范，很可能**无法通过 markdownlint check**，最终导致无法合并 PR。
+
+我们为 TiDB 中文文档提前设置了 25 条 [markdownlint 规则](/resources/markdownlint-rules.md)，并附上了简单易懂的解释，强烈推荐花 5 分钟通读一遍。
+
+假如你提 PR 之前没有熟悉相关 Markdown 规范，提 PR 时遇到了 markdownlint check 失败，也不必担心，报错信息中有错误详情、出错的文件和位置，帮你快速定位和解决问题。
+
+此外，你还可以选择在本地进行 markdownlint check：
+
+```bash
+./scripts/markdownlint [FILE...]
+```
+
+</details>
+
+<details>
+<summary>常用 bot 命令</summary>
+
+我们为 docs 和 docs-cn 仓库提前设置了一些命令语句，只要按照一定的格式在 PR 中留言，就能触发 bot 完成相应操作。详情见下表。
+
+| 命令 | 含义 | 示例 |
+| ------ | ------ | ------ |
+| `/label` | 给 PR 添加 label，多个 label 间需要用逗号分隔。如果 label 中有斜线 `/`，则命令为 `/[label 的第一个单词] [label 其他部分]` | `/label contribution`，`/translation from-docs` |
+| `/remove-label` | 删除 PR label。如果 label 中有斜线 `/`，则命令为 `/remove-[label 的第一个单词] [label 其他部分]` | `/remove-label contribution`，`/remove-translation welcome` |
+| `/assign` | 将 PR 分配给指定的人，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。如果想要将 PR 分配给自己，`/assign`后可不跟 GitHub ID。 | `/assign @lilin90` |
+| `/unassign` | 移除 PR 之前指定的 assignee。 | `/unassign @lilin90` |
+| `/cc` | 将 PR 分配给指定的 reviewer，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。 | `/cc @lilin90, @hfxsd` |
+| `/uncc` | 移除 PR 之前指定的 reviewer。  | `/uncc @lilin90`|
+
+</details>
+
+其他快速上手资源
+
+- [PingCAP 中英术语表](https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC)
+- [GitHub Docs](https://docs.github.com/en)
+- [Pull Request Commit Message 规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#how-to-write-a-good-commit-message)
+- [Pull Request 标题规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#pull-request-title-style)
+- [代码注释规范](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
+- [Figma 快速上手教程](/resources/figma-quick-start-guide.md)
+
 ## 联系我们
 
 欢迎来 [TiDB Internals](https://internals.tidb.io/) 参与讨论。
```

Comment:
TiDB Internals is the hub for discussing everything related to the implementation and design of the TiDB projects, not so appropriate for docs discussion.
```suggestion
```